Airports/EGLM/Positions.txt

```diff
@@ -0,0 +1 @@
+EGLM_R_TWR:Waltham Radio:122.600:LMR:R:EGHR:TWR:-:-:::N051.30.02.000:W000.46.29.000::
```

Comment:
```suggestion
EGLM_R_TWR:Waltham Radio:122.600:LMAG:R:EGLM:TWR:-:-:::N051.30.02.000:W000.46.29.000::
```
For 'radio' stations we use the identifier `__AG` as in Air-Ground, to avoid confusion with Radar positions :)

Reply:
got it, thanks